server/app/routes/user/index.js

```diff
@@ -50,6 +32,52 @@ router.post('/', function(req, res, next) {
     }
 })
 
+// Get a user's info, including orders, addresses, billing for personal info page
+router.use('/info', function (req, res, next) {
```

Comment:
will check if req.user exists. if it does good. if it doesn't, send back nothing.